tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -2383,44 +2375,29 @@ private:
    /// is the address pointing to the storage of the read/created object in/by
    /// the TTreeReaderValue/TemporaryBranch
    template <typename... ColumnTypes>
-   RResultPtr<RInterface<RLoopManager>> SnapshotImpl(std::string_view treename, std::string_view filename,
+   RResultPtr<RInterface<RLoopManager>> SnapshotImpl(std::string_view fullTreeName, std::string_view filename,
```

Comment:
Very minor detail: Doxygen still says `treename`

Reply:
removed the doxygen docs altogether, they did not refer to the current implementation